### ISSUE TITLE
Log out functionality is missing

### ISSUE THREAD

**Issue by rubo-21** [overtake/telegram#451]
Hey there.

I'm using telegram for macOS v 2.96.94514 
and can't find log-out button, it used to be in Edit menu but after update I can't find it.
Also I checked settings. If is missed something could you pleae tell me how to log out? 

Thanks.

**Comment by overtake** [comment]
@rubo-21 settings->edit->scroll to bottom -> logout

**Comment by rubo-21** [comment]
@overtake Thank you, anyway its very strange UX. I think it would be better to make logout more easier and logical.

BTW: why you removed logout functionality from Edit menu?

Thanks again

**Comment by KaranErry** [comment]
I agree completely with @rubo-21. When I press the "Edit" button, only the Name fields change to input fields, so the assumption becomes that there's nothing more to look for. The reason I'm on here is actually because I had to google how to Logout, because it was so hard to find. 

I think the current functionality's just stupid and frustrating, keeping the Logout button so hidden.
